src/components/imagePreviewer/index.jsx

```
@@ -0,0 +1,201 @@
+    if (!isTop) {
+      getAllLikes();
+      getAllComments();
+    }
+  }, [image?.id]);
+
+  return (
+    <>
+      <Modal title={image?.name} isOpen={isOpen} onClose={onClose} size="lg">
+        {/* Image Previewer */}
+        <div className="player">
+          <img src={image?.image} className="w-full h-full" />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)